collectRatesData.js

```diff
@@ -0,0 +1,53 @@
+const axios = require('axios')
+
+const config = require('./config');
+const helpers = require('./helpers');
+
+const logger = helpers.logger
```

Comment:
More elegant to write it like this:
`const { logger } = require('./helpers')`

Reply:
thanks, remove the logger component.